File: PC/DataDrop.Models/FileInformation.cs
using System;
using System.Collections.Generic;
using Newtonsoft.Json;

namespace DataDrop.Models
{
    public class FileInformation
    {
        [JsonProperty]
        public string Filename { get; set; }

        [JsonProperty]
        public string FileExtension { get; set; }

        [JsonProperty]
        public int FileSize { get; set; }

        [JsonProperty]
        public int BufferSize { get; set; }

        [JsonProperty]
        public int SequenceCount { get; set; }

        [JsonProperty]
        public int HashSequenceCount { get; set; }
    }
}

File: Android/app/src/main/java/at/if19b150/datadrop/FileInformation.kt
package at.if19b150.datadrop

import org.json.JSONArray

class FileInformation(var Filename : String, var FileExtension : String, var FileSize : Int, var BufferSize : Int, var SequenceCount : Int, var HashSequenceCount : Int?) {
    var FileData = mutableListOf<ByteArray?>()

    var HashSequenceList = mutableListOf<String>()

}